src/main/java/it/pagopa/pn/bff/rest/InfoRecipientController.java

```diff
@@ -42,4 +45,28 @@ public Mono<ResponseEntity<Flux<BffPgGroup>>> getPGGroupsV1(String xPagopaPnUid,
         log.logEndingProcess("getPGGroupsV1");
         return bffPgGroups.collectList().map(groups -> ResponseEntity.status(HttpStatus.OK).body(Flux.fromIterable(groups)));
     }
+
+    /**
+     * GET bff/v1/pa-list
+     * Get the list of PA that use PN
+     *
+     * @param paNameFilter The prefix of the PA name
+     * @return The list of PA
+     */
+    @Override
+    public Mono<ResponseEntity<Flux<BffPaSummary>>> getPAListV1(String paNameFilter,
+                                                                final ServerWebExchange exchange) {
+        log.logStartingProcess("getPAListV1");
+
+        var headers = exchange.getRequest().getHeaders();
+        if (!RequestUtility.checkCxIdIsValid(headers.get("x-pagopa-pn-cx-id")) ||
+                !RequestUtility.checkCxTypeIsValid(headers.get("x-pagopa-pn-cx-type"), List.of(CxTypeAuthFleet.PF, CxTypeAuthFleet.PG))) {
+            return Mono.just(ResponseEntity.badRequest().build());
```

Comment:
Instead of badRequest use status(HttpStatus.?) and send an unauthorized (maybe 403) error